Dshop/apps/products_catalogue/tests/test_api_persistent_cart.py

```diff
@@ -53,6 +53,22 @@ def test_add(tv_product):
     assert_products_data(response.data, [tv_product], [10])
 
 
+@pytest.mark.django_db
+def test_add__make_empty_get(tv_product):
+    data = {
+        'items': [ {'product_pk': tv_product.pk, 'quantity': 10} ] 
+    }
+    api_client = APIClient()
+    add_response = api_client.post(reverse("api_cart"), data)
+    empty_response = api_client.post(reverse("api_cart_empty"))
+    get_response = api_client.get(reverse("api_cart"))
+    assert add_response.status_code == status.HTTP_201_CREATED
```

Comment:
I would put those asserts right after each call. This is because if one fails, does it make sense to run others?

Reply:
trying to conform to AAA / GWT